examples/set-labels-full-coverage/README.md

```diff
@@ -1,16 +1,16 @@
-# set-labels: Advanced Example
+# set-labels: Full Coverage Example
 
 ### Overview
 
 This example demonstrates how to declaratively run [`set-labels`] function
-to upsert labels to the `.metadata.labels` field on all resources.
+to upsert all common labels to different type of resources, as well as self-defined resource.
```

Comment:
> different type of resources, as well as self-defined resource.

Maybe `different built-in resources and CustomResourceDefinition (CRD) resources`.